### DISCUSSION TITLE
Can we publish third-party themes?

### DISCUSSION THREAD

**Discussion by zachpoblete** [microsoft/MicrosoftEdge-Extensions#96] [Q&A] [answered]
I'm confused because I haven't found any official statement that we can't publish third-party themes. Yet, I only see two third-party themes in the Add-ons Store, both published by [Stefan Van Damme](https://microsoftedge.microsoft.com/addons/search?developer=Stefan%20vd) in February 2021. Only official themes from the [Microsoft Corporation](https://microsoftedge.microsoft.com/addons/search?developer=Microsoft%20Corporation) have been published since then, and I'm wondering why.

**Comment by Bhuvana-Priya** [comment, marked as answer]
Hi @zachpoblete , 
Thank you for reaching out.
Third-party themes are not yet supported in the Edge Add-ons Store. We will keep you updated on any changes.

**Reply by zachpoblete** [reply]
Thanks for answering.

I hope third-party themes are supported soon as the third-party theme I want, [Complete Black](https://microsoftedge.microsoft.com/addons/detail/complete-black-theme-for-/ghonbmojapihnfjnnmhijdjmfmbmapkk) (because the built-in dark theme ain't dark enough for me lol), looks bad from not being updated – can't even identify the active tab when using vertical tabs. And yes, I've tried literally hundreds of dark themes from the Chrome Web Store, but they don't have vertical tabs in mind, so they look bad as well.

I made my own complete black theme that fixes these issues. I would publish it if I could!

**Reply by stefanvd** [reply]
Hi there,

It is here the developer of the [Complete Black Theme for Microsoft Edge](https://microsoftedge.microsoft.com/addons/detail/complete-black-theme-for-/ghonbmojapihnfjnnmhijdjmfmbmapkk). My theme does support the vertical tab color, see my GitHub:
[https://github.com/turnoffthelights/Browser-Themes/blob/main/Microsoft%20Edge/Complete%20Black%20Theme%20for%20Microsoft%20Edge/manifest.json](https://github.com/turnoffthelights/Browser-Themes/blob/main/Microsoft%20Edge/Complete%20Black%20Theme%20for%20Microsoft%20Edge/manifest.json)

So if there is an issue, please let me know at [https://www.turnoffthelights.com/support](https://www.turnoffthelights.com/support)

Thanks,
Stefan

**Reply by zachpoblete** [reply]
Hi @stefanvd,

Thanks for letting me know there's an updated version of your theme. It does indeed fix the vertical active tab by highlighting it. Why haven't you updated the theme in the Add-ons Store to 1.0.6.0, the new version?

**Reply by stefanvd** [reply]
Hi there,

My updated theme is on the Microsoft review side. However, due to their policy, they revoked my update.

Thanks,
Stefan
